RLTest/__main__.py

```diff
@@ -537,6 +541,7 @@ def __init__(self):
         Defaults.enable_debug_command = True if self.args.allow_unsafe else self.args.enable_debug_command
         Defaults.enable_protected_configs = True if self.args.allow_unsafe else self.args.enable_protected_configs
         Defaults.enable_module_command = True if self.args.allow_unsafe else self.args.enable_module_command
+        Defaults.redis_config_file = self.args.redis_config_file if self.args.redis_config_file else None
```

Comment:
Does this new arg make sense in all envs?

Reply:
Can a simple assignment be enough?

Reply:
It could make sense, but for the current cases, the use of the same file is enough.